### PR TITLE
feature: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+msgstr "Minimalus pralaidumas"
+
+msgid "Minimum intensive bandwidth"
+msgstr "Minimalus intensyvus pralaidumas"
+
+msgid "Minimum rendering time"
+msgstr "Minimalus atvaizdavimo laikas"
+
+msgid "Missing field %s"
+msgstr "Trūkstamas %s laukas "
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: additional space on the end of sentence?

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: so in the translation source files

Comment (bartlomiejmarszal) [reply locales/lt-LT/messages.po]: In the translation file, there is no space at the end of the sentence.

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: image

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

**Pull request merged**